### DISCUSSION TITLE
Colour scheme

### DISCUSSION THREAD

**Discussion by jeffrey-tafe** [gh44922023/Tafe230221#2] [General]
Do we want to use a common colour scheme across the application or will each calculator and the main page be coloured separately?

**Comment by BrodieBrown1998** [comment]
Use a light blue colour scheme.

**Reply by jeffrey-tafe** [reply]
I have grabbed the hex values from the main menu colour scheme and used them in the trip calculator. Does that work for you @gh44922023 ?

**Comment by gh44922023** [comment]
Yes I agree, we use the main menu colour scheme for buttons and output for project consistency.
